core/crypto/bip39/mnemonic.hpp

```diff
@@ -29,12 +29,12 @@ namespace kagome::crypto::bip39 {
   struct Mnemonic {
     using Words = std::vector<std::string>;
 
-    boost::variant<common::Buffer, Words> seed;
+    std::variant<Bip39Seed, Words> seed;
```

Comment:
Must be `variant<vector, Words>` or `variant<byte[32], Words>`,
it's not `Bip39Seed = byte[64]`